.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] I can see that you did a great job on this project so far but you still need to work on something for your project to match all the requirements, I noticed that you forget to Ignore also the `test.md` file , kindly fix this :wink: :wink: